docs/copilot/copilot-chat.md

```diff
@@ -57,6 +57,12 @@ The following example shows the `preview` mode for inline chat.
 
 ![Copilot inline chat preview mode, showing the code changes side-by-side](images/copilot-chat/inline-chat-preview-mode.png)
 
+## Experimental terminal inline chat
```

Comment:
While this is closely related to inline chat, it feels wrong to feature this before the chat view. @ntrogh any thoughts on where we should stick this? We could also hold off on adding a section until it's non-experimental.

Reply:
Yeah I agree. I wasn't sure about this.

Reply:
We typically hold off on putting experimental features in the main docs.